43_tpm_1_2_support.md

```diff
@@ -0,0 +1,176 @@
+# enhancement-43: Extend support for TPM 1.2
+
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Notes/Constraints/Caveats (optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed (optional)](#infrastructure-needed-optional)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+
+- [ ] Enhancement issue in release milestone, which links to pull request in [keylime/enhancements]
+- [ ] Core members have approved the issue with the label `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place
+- [ ] User-facing documentation has been created in [keylime/keylime-docs]
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+## Summary
+
+This enhancement proposes to extend support for TPM 1.2 which was removed in Keylime v6.0.0
+
+
+## Motivation
+
+Modern servers usually have TPM 2.0 chip rather than TPM 1.2, however not all vendors support 
+measured boot with TPM 2.0 in legacy boot mode which means that the latest Keylime software 
+cannot be used for remote attestation of those legacy systems. 
+Extended support for TPM 1.2 will allow Keylime users to use the latest Keylime versions 
+without backporting patches for potential vulnerabilities that may arise in the future. 
+With the enhancement the same Keylime baseline can be used for remote attestation of both
+TPM 1.2 and TPM 2.0 systems which will make the future migration process from TPM 1.2 to TPM 2.0
+more smooth.
+
+
+### Goals
+
+Add TPM 1.2 support on master branch.
+
+### Non-Goals
+
```

Comment:
The commit that removed the TPM 1.2 support from Keylime also removed the "deep quote" functionality because it was flawed and not very useful. So it would probably be a good idea to mention that resurrecting this code won't restore the deep quote stuff.